### PR TITLE
SVRCOM-2208 Include statement on Functions GA in release notes, remove TP warnings

### PR DESCRIPTION
**Pull request by gabriel-rh**
Version(s):
4.8+

Issue:
https://issues.redhat.com/browse/SRVCOM-2208

Link to docs preview:
https://53841--docspreview.netlify.app/openshift-enterprise/latest/serverless/serverless-release-notes.html#serverless-rn-1-26_serverless-release-notes

QE review not required

### CODE REVIEW

**Review by rolfedh — APPROVED** [review]
LGTM

File: modules/serverless-rn-1-26-0.adoc [hunk 1]
```diff
@@ -11,6 +11,7 @@
 [id="new-features-1.26_{context}"]
 == New features
 
+* {FunctionsProductName} is now GA.
```
Comment (rolfedh) [inline modules/serverless-rn-1-26-0.adoc:14]: Outside the scope of this PR: I see we're using the term, "GA", widely. How might we ensure that its meaning is clear to all users?

Comment (rolfedh) [reply modules/serverless-rn-1-26-0.adoc:14]: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes

Comment (gabriel-rh) [reply modules/serverless-rn-1-26-0.adoc:14]: This is a good point. I looked at other instances in our docs and the preferred representation is "General Availability (GA)", but the Serverless style seems to be just "GA" so I went with that here but feet the other way is better. 

I agree that we need to look at this to make it consistent across products.

Comment (rh-max) [reply modules/serverless-rn-1-26-0.adoc:14]: I suppose that for the Serverless the reason for shortened "GA" is that it is expanded in the top "Generally Available and Technology Preview features" section of the Release Notes.

**Pull request merged**